.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           version: latest
 
       - name: Start Supabase local development setup
-        run: supabase start --workdir infra -x studio,inbucket,edge-runtime,logflare,vector,supavisor,imgproxy,storage-api
+        run: make run_infra
```

Comment:
If we're making this change then we should remove the `Install Supabase CLI` step above it since `npx` handles the install for us.